roles/openshift_certificate_expiry/README.md

```diff
@@ -104,35 +139,71 @@ $ ansible-playbook -v -i HOSTS /usr/share/ansible/openshift-ansible/playbooks/ce
 
 > [View This Playbook](../../playbooks/certificate_expiry/easy-mode.yaml)
 
-***
+### Easy mode and upload reports to masters
+
+This example builds on top of [easy-mode.yaml](#easy-mode) and additionally
+uploads a copy of the generated reports to the masters, with a timestamp in the
+file names.
+
+This is specially useful when the playbook runs from within a container, because
+the reports are generated inside the container and we need a way to access them.
+Uploading a copy of the reports to the masters is one way to make it easy to
+access the reports. Alternatively you can use the
+[role variables](#role-variables) that control the path of the generated reports
+to point to a container volume.
 
-Default behavior:
+With the container use case in mind, this playbook allows control over some
+options via environment variables:
+
+ - `CERT_EXPIRY_WARN_DAYS`: sets `openshift_certificate_expiry_warning_days`,
```

Comment:
On one hand I wanted to show how to change the default - I could do this by setting the value to the same as the default, but I think it looks more clear if the value differs.

But more importantly, I believe that the default of 30 might be a bit too short in some cases. For example: this playbook is used in the [scheduled certificate check example](../blob/cronjob-example/examples/scheduled-certcheck-upload.yaml) which has a default schedule of "every 1st day of the month", which I think is reasonable. If your CA cert expires on April 1st, the check that runs on March 1st with a default window of 30 days won't warn you, and chances are that you'll have a fun April Fools' day... 

Even in less extreme cases, I believe cluster admins prefer to have a fairly advanced warning in order to have time to plan cert redeployment. In fact I was wondering if the role's default should be increased, with this example's value set even higher :)   What do you think?

Slightly related to this, I think we could improve the role by allowing env var overrides on all defaults - this would allow runtime customization without the need to touch playbooks, which would be very welcome in the container use case... but that's beyond scope of this PR, so if you're ok with that I'll create a separate issue/PR.